README.rst

```diff
@@ -72,12 +72,27 @@ Usage
   .. code:: python
 
     def test_explicitly_enable_socket(socket_enabled):
-        assert socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        assert socket.socket.connect(('127.0.0.1', 80))
 
 
     @pytest.mark.enable_socket
     def test_explicitly_enable_socket_with_mark():
-      assert socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        assert socket.socket.connect(('127.0.0.1', 80))
+
+* To allow only specific hosts per-test:
+
+  .. code:: python
+
+    @pytest.mark.enable_socket(['127.0.0.1'])
+    def test_explicitly_enable_socket_with_mark():
+        assert socket.socket.connect(('127.0.0.1', 80))
+
+or for whole test run
+
+  .. code:: ini
+
+    [pytest]
+    addopts = --disable-socket=127.0.0.1,127.0.1.1
```

Comment:
So the syntax doesn't really make sense to me - here it reads that I'd like to disable socket on these hosts, whereas it appears that passing the list actually enables socket on those hosts.

Should there be a separate flag that takes the hosts list like:
```
pytest --disable-socket --enable-socket-hosts=127.0.0.1,127.1.0.1
```

Reply:
yea i had thought the same while i was doing it. possibly even a single command line argument `--only-allow-hosts=127.0.0.1,127.1.0.1` (since `--disable-socket` becomes redundant). happy to make this change.

Reply:
added `--restrict_hosts=127.0.0.1` in https://github.com/miketheman/pytest-socket/pull/9 but need help getting it running with 2.7.